content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/docker.md

```diff
@@ -82,11 +91,17 @@ To stop producing traces, uninstall APM and restart the infrastructure:
    ```
    Or use the equivalent for your environment.
 
+## Troubleshooting
+
+If you encounter problems enabling APM with SSI, see the [SSI troubleshooting guide][17].
+
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: /tracing/trace_collection/library_config/
+[3]: /tracing/trace_collection/library_config/
```

Comment:
```suggestion
[3]: /tracing/trace_collection/library_config/
```
I think this accidentally got duplicated as this should link to environment variables ?